### ISSUE TITLE
SQLite Database error when running from Docker image

### ISSUE THREAD

**Issue by jacksgt** [drone/drone#1991]
Hi,

I wanted to start experimenting with Drone and downloaded the Docker image. According to the [docs](http://readme.drone.io/admin/database-engines/), database configuration is not required as Drone will use its built-in SQLite database by default.

However:
```shell
$ docker run -it drone/drone
Unable to find image 'drone/drone:latest' locally
latest: Pulling from drone/drone
a3ed95caeb02: Pull complete 
802d894958a2: Pull complete 
eb9bbc802fed: Pull complete 
Digest: sha256:c361a2da847834ba91fa53ac32681ace311bf578c0f558445c530ece7a247376
Status: Downloaded newer image for drone/drone:latest
ERRO[0030] unable to open database file                 
FATA[0030] database ping attempts failed
```
I also tried the the image `drone/drone:0.6`, as it seems to be more recent than the `latest` tag.
```shell
$ docker run -it drone/drone:0.6
Unable to find image 'drone/drone:0.6' locally
0.6: Pulling from drone/drone
a3ed95caeb02: Already exists 
802d894958a2: Already exists 
6a7d3a472b1d: Pull complete 
Digest: sha256:19cf468b904661d96372bd39def53de6f6f192ecd7394ca9cfced362d3e6e7cb
Status: Downloaded newer image for drone/drone:0.6
ERRO[0030] unable to open database file                 
FATA[0030] database ping attempts failed
```

I also tried manually setting `DRONE_DATABASE_DRIVER` (via env var) to `sqlite3`, but I still got the same error message.

**Comment by bradrydzewski** [comment]
https://gitter.im/drone/drone for support questions

**Comment by bradrydzewski** [comment]
tldr you need to mount a volume

**Comment by jacksgt** [comment]
Thanks for the very helpful comment...

Anyway, it appears Drone does not create the the path `/var/lib/drone` itself and therefore fails if it is not present (although the error message could be more helpful in this case).

This can be worked around by either mounting a volume (`-v /tmp/drone:/var/lib/drone`) or creating the directory before running the container (e.g. by adding `RUN mkdir -p /var/lib/drone` to the Dockerfile).

**Comment by bradrydzewski** [comment]
The comment reflects the github [issue template](https://raw.githubusercontent.com/drone/drone/master/.github/issue_template.md), which politely instructs individuals to post questions to our forum before creating false positive issues. Or search for [existing issues](https://github.com/drone/drone/issues?q=is%3Aissue+unable+to+open+database+file+is%3Aclosed). Or search Google for [existing answers](https://www.google.co.kr/webhp?sourceid=chrome-instant&rlz=1C5CHFA_enUS699US700&ion=1&espv=2&ie=UTF-8#q=drone+unable+to+open+database+file&*). Or to follow the docs an mount a volume.
